Mathlib/Algebra/Lie/Solvable.lean

```diff
@@ -211,6 +217,13 @@ instance isSolvableAdd {I J : LieIdeal R L} [hI : IsSolvable R I] [hJ : IsSolvab
   exact ⟨⟨k + l, LieIdeal.derivedSeries_add_eq_bot hk hl⟩⟩
 #align lie_algebra.is_solvable_add LieAlgebra.isSolvableAdd
 
+theorem derivedSeries_ne_top_of_solvable [IsSolvable R L] [Nontrivial L] :
+    derivedSeries R L 1 ≠ ⊤ := by
```

Comment:
I think it is slightly more useful to prove:
```suggestion
theorem derivedSeries_lt_top_of_solvable [IsSolvable R L] [Nontrivial L] :
    derivedSeries R L 1 < ⊤ := by
```
since it is _slightly_ more easier to pass from `I < J` to `I ≠ J` using `LT.lt.ne` than vice-versa (when `J = ⊤`).